packages/sommelier.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Sommelier < Package
+  description 'Sommelier works by redirecting X11 and Wayland programs to the built-in ChromeOS wayland server.'
+  homepage 'https://chromium.googlesource.com/chromiumos/containers/sommelier'
+  version '1382c'
+  source_url 'https://chromium.googlesource.com/chromiumos/containers/sommelier/+/0.20/README?format=TEXT'
+  source_sha256 'b58d799b16d20abf92369fe0749c73f7398996f0afa9933517051778a8bb16c3'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'mesa'
+  depends_on 'pixman'
+  depends_on 'libxkbcommon'
+  depends_on 'xwayland'
+  depends_on 'xkbcomp'
+  
+  def self.build
+    system 'git clone https://chromium.googlesource.com/chromiumos/containers/sommelier'
+    Dir.chdir ("sommelier") do
+      system "git reset --hard 1382ce084cc40790340d672e8b62ec47733cb860"
```

Comment:
This could also be `system 'git checkout 59dd453e6efd2c4a0b2b08767c42f47e66a4aaa1'`.  Why are you not picking the latest hash from master?

Reply:
No. The latest hash has some incompatible code. Git gives me `Unknown ref 59dd453e6efd2c4a0b2b08767c42f47e66a4aaa1` when I try the git checkout thing.

Reply:
What is incompatible?  Just trying to understand why we need an earlier commit.

Reply:
It has a makefile specific to Google that isn't open source, and it requires a header file that Crew doesn't have.

Reply:
Look at the git log: `gyp` `virtwl` https://chromium.googlesource.com/chromiumos/containers/sommelier

Reply:
Can we add those packages?

Reply:
No. Virtwl is a linux header file, and I can't find gyp anywhere. More trouble than it's worth to fix something that's not broken.

Reply:
This is an interesting commit a few down from the latest master commit.
```
commit d0a225e2af73483aff02ab9ddb8a710f56ae4349
Author: David Reveman <reveman@chromium.org>
Date:   Thu Apr 5 23:14:05 2018 +0000
    
    Remove libsystemd dependency and add --sd-notify flag.
```